examples/kivy/main.py

```diff
@@ -41,7 +41,9 @@ async def example(self):
         while self.running:
             try:
                 self.line("scanning")
-                scanned_devices = await bleak.BleakScanner.discover(1)
+                scanner = bleak.BleakScanner()
+                await scanner.request_permissions()
+                scanned_devices = scanner.discover(1)
```

Comment:
`discover()` is not an instance method, so we should not be creating a `BleakScanner()` instance.

Reply:
done